### PR TITLE
Support new solc and truffle

### PR DESCRIPTION
**Pull request by gleim**
Add new independent sub-directory for upgraded Tokens

### CODE REVIEW

File: Token/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+README.md
+===
+
+This folder contains an updated version of Token/Token_Contracts/ that is compliant with:
+  - solc       0.4.2
```
Comment (ethers) [inline Token/README.md]: may as well use 0.4.4

File: Token/contracts/HumanStandardToken.sol [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Imagine coins, currencies, shares, voting weight, etc.
+Machine-based, rapid creation of many tokens would not necessarily need these extra features or will be minted in other manners.
+
+1) Initial Finite Supply (upon creation one specifies how much is minted).
+2) In the absence of a token registry: Optional Decimal, Symbol & Name.
+3) Optional approveAndCall() functionality to notify a contract if an approval() has occurred.
+
+.*/
+
+pragma solidity ^0.4.2;
```
Comment (ethers) [inline Token/contracts/HumanStandardToken.sol]: May as well lock to `pragma solidity 0.4.4;`

**Comment by gleim** [comment]
Will do.

**Comment by simondlr** [comment]
Hey @gleim. Thanks for this. Check out https://github.com/ConsenSys/Tokens/pull/37. Think it would be better to have separate PRs for Zeppeling related work?

**Comment by gleim** [comment]
@simondlr yes, don't recall how they initially became conflated

**Comment by skmgoldin** [comment]
What do we want to do with this PR? What exactly is the idea behind it?

**Comment by simondlr** [comment]
Initially, it was just pull in new solc and truffle, but got conflatd. I think we can close this for now. :)

**Pull request closed without merging**